cmd/snap/cmd_changes.go

```diff
@@ -63,6 +63,17 @@ func (s changesByTime) Swap(i, j int)      { s[i], s[j] = s[j], s[i] }
 
 var allDigits = regexp.MustCompile(`^[0-9]+$`).MatchString
 
+func queryChanges(cli *client.Client, opts *client.ChangesOptions) ([]*client.Change, error) {
```

Comment:
Same question as above. Perhaps the rebooting part should be in a helper?

Reply:
> Same question as above. Perhaps the rebooting part should be in a helper?

This is precisely that helper, it seems? The code you commented earlier is what makes Changes return the error.